ui/src/vertical-geometry/displayed-position-guide.tsx

```diff
@@ -49,7 +42,7 @@ export const DisplayedPositionGuide: React.FC<DisplayedPositionGuideProps> = ({
         guideRectangleMinWidthPx,
     );
     const guideRectangleStartPx =
-        guideStartPx + guideWidthPx * (displayedMetersAtLeftEdge / maxDisplayedMeters);
+        guideStartPx + guideWidthPx * (displayedMetersAtLeftEdge / maxMeters);
```

Comment:
Just in case pitäiskö tsekata onko maxMeters <= 0?

Reply:
Lisäilin chekkauksen.

Ainahan tällainen on vähän pallottellua sen puolesta, että miten defensiivistä koodia halutaan (ja miten syvälle nuo tarkistukset kaivetaan). Tässä tapauksessa ihan ok bongaus varmistaa että tuo on positiivinen, kun oletusarvoksikin nakataan nolla: 
https://github.com/finnishtransportagency/geoviite/blob/8afb61d645e3dd6eb2c3b2483cd4e2f29ea1b31e/ui/src/vertical-geometry/vertical-geometry-diagram-holder.tsx#L143-L146

Joskin kun kokeilin mitä kuvaajalle tapahtuu, kun tuon endin arvoksi päätyy 0 tai -1, niin kuvaajan sijaan näytetään vain virheviesti "Valitun raiteen pystygeometriaa ei voida esittää.". Renderöinti ei tällaisessa tilanteessa siis edes päätynyt tämän pullarin komponentille asti, eli sinänsä chekkauksella ei ollutkaan nykytilanteessa merkitystä 🤷 .